distribution/helm/vector-agent/templates/daemonset.yaml

```diff
@@ -13,10 +13,11 @@ spec:
     type: {{ .Values.updateStrategy }}
   template:
     metadata:
-    {{- with .Values.podAnnotations }}
       annotations:
-        {{- toYaml . | nindent 8 }}
-    {{- end }}
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        {{- with .Values.podAnnotations }}
+          {{- toYaml . | nindent 8 }}
+        {{ end }}
```

Comment:
```suggestion
        {{- end }}
```

Reply:
Dang DCO, and doing things from my phone.